modules/porous_flow/doc/content/source/materials/PorousFlowHystereticRelativePermeabilityGas.md

```diff
@@ -0,0 +1,29 @@
+# PorousFlowHystereticRelativePermeabilityGas
+
+This Material computes the gas relative permeability using the van-Genuchten formulation.  The relative permeability is assumed to be hysteretic.  This Material may be used two-phase situations only.
```

Comment:
```suggestion
This Material computes the gas relative permeability using the van Genuchten formulation.  The relative permeability is assumed to be hysteretic.  This Material may be used in two-phase situations only.
```

Reply:
What is it with me a van-HYPHEN-Genuchten ?!?!